app/src/main/java/com/madonasyombua/sportsdb/ui/MainActivity.kt

```diff
@@ -1,15 +1,47 @@
 package com.madonasyombua.sportsdb.ui
 
-import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
-import com.madonasyombua.sportsdb.R
+import androidx.activity.compose.setContent
+import androidx.activity.viewModels
+import androidx.appcompat.app.AppCompatActivity
+import androidx.compose.foundation.ExperimentalFoundationApi
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Surface
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.tooling.preview.Preview
+import com.madonasyombua.sportsdb.ui.screen.home.HomeViewModel
+import com.madonasyombua.sportsdb.ui.screen.home.HomeScreen
+import com.madonasyombua.sportsdb.ui.screen.teams.TeamsViewModel
+import dagger.hilt.android.AndroidEntryPoint
 
+@AndroidEntryPoint
 class MainActivity : AppCompatActivity() {
-
+    private val viewModel: HomeViewModel by viewModels()
+    private val teamsViewModel:TeamsViewModel by viewModels()
+    @ExperimentalFoundationApi
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_main)
+        setContent {
+            Surface() {
+                MyApp(viewModel,teamsViewModel = teamsViewModel)
+            }
+        }
 
     }
 
 }
+
+@ExperimentalFoundationApi
+@Composable
+fun MyApp(viewModel: HomeViewModel,teamsViewModel: TeamsViewModel) {
+    HomeScreen(viewModel,teamsViewModel = teamsViewModel)
+
+}
+
+@Preview("Light Theme", widthDp = 360, heightDp = 640)
+@Composable
+fun LightPreview() {
+    MaterialTheme {
+        // MyApp()
```

Comment:
What is `//MyApp()`

Reply:
for previewing, will set it up with dummy data

Reply:
Great